etc/scripts/clean-jupyterlab.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ANACONDA_HOME=${ANACONDA_HOME:='~/opt/anaconda'}
+JUPYTERLAB_HOME=${JUPYTERLAB_HOME:='~/opensource/jupyterlab/jupyterlab'}
+
+LAB_VERSION="1.2.13"
+
+if [ "$1" = "--version" ]
+then
+    LAB_VERSION="$2"
+fi
+LAB_VERSION="==$LAB_VERSION"
+
+echo "Anaconda..........: $ANACONDA_HOME"
+echo "Anaconda env......: $CONDA_DEFAULT_ENV"
+echo "JupyterLab........: $JUPYTERLAB_HOME"
+echo "JupyterLab Version: $LAB_VERSION"
+echo " "
+
+set -e
+pip uninstall -y elyra || true;
+pip uninstall -y jupyterlab-git || true;
+pip uninstall -y jupyterlab-server || true;
+pip uninstall -y jupyterlab || true;
+pip uninstall -y nbdime || true;
+
+# Clean jupyterlab environment / workspace
+echo "Cleaning jupyter and jupyterlab environment / workspace"
```

Comment:
Given this cleans both base and current env we should make this echo more detailed. It should clearly state that it is cleaning both `base` and `$CONDA_DEFAULT_ENV` env